content/en/tracing/setup/dotnet.md

```diff
@@ -282,7 +282,7 @@ The .NET Tracer can instrument the following libraries automatically:
 
 | Framework or library           | NuGet package name                       | Package versions     | Integration Name     |
 |--------------------------------|------------------------------------------|----------------------|----------------------|
-| ASP.NET MVC                    | `Microsoft.AspNet.Mvc`                   | 5.1.0+               | `AspNetMvc`          |
+| ASP.NET MVC                    | `Microsoft.AspNet.Mvc`                   | 4.0-5.0+               | `AspNetMvc`          |
```

Comment:
@andrewsouthard1: I think we can just say `4.0+` which means "4.0 and above" and includes the `5.x` versions.

Alternatively, we can be super specific and say `4.0.40804 - 5.2.7` since we haven't really tested `4.0` versions lower than that one.

Your pick!

Reply:
Seconding the above. Would also like this to be `4.0+`